release/nightly_tests/dataset/tpch_q1.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+from datetime import datetime, timedelta
+from typing import Dict
+
+import numpy as np
+import ray
+
+# TODO: We should make these public again.
+from ray.data._internal.aggregate import Count, Sum, Mean
+
+from benchmark import Benchmark
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(description="TPCH Q1")
+    parser.add_argument("--sf", choices=[1, 10, 100, 1000, 10000], type=int, default=1)
+    return parser.parse_args()
+
+
+def main(args):
+    path = f"s3://ray-benchmark-data/tpch/parquet/sf{args.sf}/lineitem"
+    benchmark = Benchmark("tpch-q1")
+
+    def benchmark_fn():
+        (
+            ray.data.read_parquet(path)
```

Comment:
Maybe add a quick comment on what we are testing for here, seems like it is just filtering down a particular slice of the data then testing aggregates on multiple columns but might be good to add why we are doing multiple columns etc.

Reply:
Ah, good call. I just directly adapted the TPC-H query code from https://examples.citusdata.com/tpch_queries.html.

Added a comment